.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-	"editor.codeActionsOnSave": { "source.fixAll.eslint": true },
+	"editor.codeActionsOnSave": { "source.fixAll.eslint": "explicit" },
```

Comment:
Ha, this has been happening on all my repos. A little inconvenient. I was planning on doing a general update sweep eventually. This works too.